examples/hosted-fungible/index.html

```diff
@@ -76,10 +76,18 @@
         <div class="description">
           <h1>Fungible</h1>
           <p>
-            This is an application tracking the transfer of fungible tokens between accounts.
+            This application is a demo to showcase personal chains, real-time transfers,
+            and trustless user interfaces in Linera.
+          </p>
+          <h4>Send funds to a friend and watch their balance update in real time.</h4>
```

Comment:
I don't think this is really a heading for the things below it.  Maybe you meant to use `<p><em>`?